lib/faker/address/es.ex

```diff
@@ -429,37 +429,44 @@ defmodule Faker.Address.Es do
   end
 
   @doc """
-  Return state.
+  Return state. But Spain doesn't has states so this calls Faker.Address.Es.region() instead
+  """
+  @spec state() :: String.t()
+  def state, do: region()
+
+  @doc """
+  Return region.[Source](https://www.ine.es/daco/daco42/codmun/cod_ccaa.htm)
 
   ## Examples
 
-      iex> Faker.Address.Es.state()
-      "Castilla-La Mancha"
-      iex> Faker.Address.Es.state()
-      "Comunidad Valenciana"
-      iex> Faker.Address.Es.state()
-      "Galicia"
-      iex> Faker.Address.Es.state()
-      "Cataluña"
+      iex> Faker.Address.Es.region()
+      "Extremadura"
+      iex> Faker.Address.Es.region()
+      "Aragón"
   """
-  @spec state() :: String.t()
-  sampler(:state, [
+
+  @spec region() :: String.t()
+  sampler(:region, [
```

Comment:
Are we cool with deviating from the other address modules? The others don’t have a region function.

Reply:
I've followed the italian version when implementing this https://github.com/elixirs/faker/blob/4ec3a668ec350e8846c786a8b3fcc8e0b9ce4a89/lib/faker/address/it.ex#L962

Reply:
I didn’t see that in the Italian translations. Thanks for pointing that out.